policyengine_us/reforms/congress/delauro/american_family_act.py

```diff
@@ -9,21 +10,20 @@ class ctc_child_individual_maximum_arpa(Variable):
         unit = USD
         documentation = "The CTC entitlement in respect of this person as a child, under the American Rescue Plan Act."
         definition_period = YEAR
+        defined_for = "ctc_qualifying_child"
 
         def formula(person, period, parameters):
             age = person("age", period)
-            is_dependent = person("is_tax_unit_dependent", period)
             amount_pre_baby_bonus = parameters(
                 period
             ).gov.irs.credits.ctc.amount.arpa.calc(age)
             # Add the baby bonus.
             baby_bonus_amount = parameters(
                 period
             ).gov.contrib.congress.delauro.american_family_act.baby_bonus
-            is_baby = age == 0
+            is_baby = age < 1
```

Comment:
should we make this a parameter?

Reply:
not now - it's really a payment at birth